hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -169,14 +175,19 @@ public URI getUri() {
     return this.uri;
   }
 
+  public String getFileSystemID() {
+    return fileSystemID;
+  }
+
   @Override
   public FSDataInputStream open(final Path path, final int bufferSize) throws IOException {
     LOG.debug("AzureBlobFileSystem.open path: {} bufferSize: {}", path, bufferSize);
     statIncrement(CALL_OPEN);
+    TracingContext tracingContext = new TracingContext(clientCorrelationID, fileSystemID, "OF");
```

Comment:
Also, tracingContext init should also take in the parameter to enableRequestCorrelations. If enableRequestCorrelations=false, then tracingContext.toString() needs to return only the clientRequestId. Else, all the fields.

Reply:
Changed. TC init takes TracingContextFormat enum, which is used in toString()